packages/models-library/src/models_library/api_schemas_payments/errors.py

```diff
@@ -0,0 +1,65 @@
+from pydantic.errors import PydanticErrorMixin
+
+
+class BaseServiceError(PydanticErrorMixin, ValueError):
```

Comment:
should be private?

Reply:
It's a good point. By making it private we are enforcing people to use more concrete exceptions in the interfaces which could be a good thing to do :-)